src/webui/webapplication.cpp

```diff
@@ -769,6 +770,25 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_rename()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hash" << "name");
+
+    QString hash = request().posts["hash"];
+    QString name = request().posts["name"].trimmed();
+
+    BitTorrent::TorrentHandle *const torrent = BitTorrent::Session::instance()->findTorrent(hash);
+    if (torrent && !name.isEmpty()) {
+        name.replace(QRegExp("\r?\n|\r"), " ");
+        qDebug() << "Renaming" << torrent->name() << "to" << name;
+        torrent->setName(name);
+    } else {
```

Comment:
### 2. If blocks ###
#### a. Multiple tests ####
```c++
if (condition) {
    //code
}
else if (condition) {
    //code
}
else {
    //code
}
```
The `else if`/`else` must be on their own lines.